code/modules/lighting/lighting_system.dm

```diff
@@ -131,8 +151,8 @@
 #else
 	distance = max(abs(x - L.__x), abs(y - L.__y))
 #endif
-	return ( L.cap ? L.cap : LIGHTING_CAP ) * (L.radius - distance) / L.radius
-//LIGHTING_CAP == strength for now
+
+	return Clamp(L.cap * (L.radius - distance) / L.radius, 0, LIGHTING_CAP)
```

Comment:
ehh, I'd rather not do that since a var gets defined in this proc. this would fail to compile in any proc that declares a var `distance` if it were defined as a macro.